src/app/address/controllers/addressResults.js

```diff
@@ -15,23 +15,19 @@ class AddressResultsController extends BaseController {
         const allAddresses = req.sessionModel.get("searchResults");
 
         const choosenAddress = allAddresses.find(
-          (address) => address.label === selectedAddress
+          (address) => address.text === selectedAddress
         );
 
-        const address = {
-          addressLine1: choosenAddress.buildingNumber,
-          addressLine2: choosenAddress.streetName,
-          addressTown: choosenAddress.town,
-          addressPostcode: choosenAddress.postcode,
-        };
+        delete choosenAddress.value;
+        delete choosenAddress.text;
```

Comment:
We delete the pretty print values to match what I persume the expected object will be when I post back to the BE